### PR TITLE
COMMERCE-9728 Support ProductSubscriptionConfiguration in DTO for Pro…

### PR DESCRIPTION
**Pull request by brianikim**
…duct.

@andrea-ale-sbarra ,

We already support post/patch/get for this. We just need to return in DTO .

Sincerely,
Brian I. Kim

### CODE REVIEW

**Comment by brianikim** [comment]
ci;test:sf

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java [hunk 1]
```diff
@@ -170,6 +173,35 @@ private CPType _getCPType(String name) {
...
+				enable = cpDefinition.isSubscriptionEnabled();
+				length = cpDefinition.getSubscriptionLength();
+				numberOfLength = cpDefinition.getMaxSubscriptionCycles();
+				subscriptionType =
+					ProductSubscriptionConfiguration.SubscriptionType.create(
+						cpDefinition.getSubscriptionType());
+				subscriptionTypeSettings =
+					cpDefinition.getSubscriptionTypeSettingsUnicodeProperties();
+			}
+		};
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java:202]: this is not needed this is already supported via NestedFields

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
I'll close this PR and move the ticket to QA check because this was already fixe by COMMERCE-11462

**Pull request closed without merging**